src/Install.php

```diff
@@ -379,6 +379,7 @@ public static function create_events() {
 		if ( ! wp_next_scheduled( 'wc_admin_daily' ) ) {
 			wp_schedule_event( time(), 'daily', 'wc_admin_daily' );
 		}
+		// @todo This is potentially redundant when the core package exists.
```

Comment:
Could this be conditional on a `wp_next_scheduled()` check?

Reply:
I don't think `wp_next_scheduled()` will work here because it's not a recurring event.

Reply:
It could stop queuing two events, but if the first has already run and completed - we'd schedule another one.

Reply:
Better for a follow up then.